src/polartoolkit/fetch.py

```diff
@@ -4256,10 +4256,14 @@ def ghf(
     From :footcite:t:`stalantarctic2021` and :footcite:t:`stalantarctic2020a`.
     Accessed from https://doi.pangaea.de/10.1594/PANGAEA.924857
 
+    version='HR24'
+    From :footcite:t:`hazzardantarctic2024`.
+    Accessed from https://osf.io/54zam/overview
+
     Parameters
     ----------
     version : str
-        Either 'burton-johnson-2020', 'losing-ebbing-2021', 'aq1',
+        Either 'burton-johnson-2020', 'losing-ebbing-2021', 'aq1', 'HR24',
```

Comment:
just adding some other missing version names.
```suggestion
        Either 'an-2015', 'martos-2017', 'shen-2020', 'burton-johnson-2020', 'losing-ebbing-2021', 'aq1' or 'HR24',
```